HappyAnding/HappyAnding/Views/ExploreShortcutViews/ExploreShortcutView/CategoryView.swift

```diff
@@ -30,31 +31,34 @@ struct CategoryView: View {
                     .frame(maxWidth: .infinity, alignment: .leading)
                 
                 Spacer()
-                
-                NavigationLink(destination: ListCategoryView()) {
+                                
+                Button(action: {
+                    self.isTappedPlutButton.toggle()
+                }, label: {
                     Text(isTappedPlutButton ? "펼치기" : "접기")
                         .Footnote()
                         .foregroundColor(Color.Gray4)
                         .padding(.trailing, 16)
-                        .onTapGesture {
-                            self.isTappedPlutButton.toggle()
-                        }
-                }
+                })
             }
             .padding(.leading, 16)
             LazyVGrid(columns: [GridItem(.flexible()), GridItem(.flexible())]) {
                 ForEach(Array(Category.allCases.enumerated()), id: \.offset) { index, value in
                     if index < categoryIndex {
-                        NavigationLink(destination:
-                                        ShortcutsListView(shortcuts: $shortcutsZipViewModel.shortcutsInCategory[value.index], categoryName: value)
-                        ) {
+                        NavigationLink(value: value, label: {
                             CategoryCellView(categoryName: value.translateName())
+                        })
+                        .navigationDestination(for: Category.self) { category in
+                            ShortcutsListView(shortcuts: $shortcutsZipViewModel.shortcutsInCategory[category.index],
+                                              categoryName: category,
+                                              navigationParentView: .shortcuts)
                         }
                     }
                 }
             }
             .padding(.horizontal, 16)
```

Comment:
카테고리뷰 링크관련 커밋이 빠졌나봐요 ..!
```suggestion
                        NavigationLink(value: value, label: {
                            CategoryCellView(categoryName: value.translateName())
                        })
                    }
                }
            }
            .padding(.horizontal, 16)
            .navigationDestination(for: Category.self) { category in
                ShortcutsListView(shortcuts: $shortcutsZipViewModel.shortcutsInCategory[category.index],
                                  categoryName: category,
                                  navigationParentView: .shortcuts)
            }
```

Reply:
감사해요 그린 !! 👍👍